Makefile

```diff
@@ -269,6 +269,9 @@ rpm:
 	mkdir -p $(RPMBUILDDIR)/SPECS
 	sed -e 's/@version@/$(PACKAGE_VERSION)/g; s/@release@/$(PACKAGE_RELEASE)/g' contrib/centos/aziot-identity-service.spec >$(RPMBUILDDIR)/SPECS/aziot-identity-service.spec
 
+	# Copy preset file to be included in the package
+	cp contrib/centos/00-aziot.preset $(RPMBUILDDIR)
```

Comment:
@arsing I wasn't exactly sure where to put this; if you know a more suitable place, please suggest.

Reply:
Extra source files like patches or this go in SOURCES/, next to the regular tarball.

Reply:
Ok, moved it.